TwoPageEBookReader/build.gradle

```diff
@@ -20,9 +20,9 @@ allprojects {
         mavenCentral()
 
         //This repo is used to define a specific snapshot version of the library you want to use
-        maven {
-            url 'https://androidx.dev/snapshots/builds/7633395/artifacts/repository'
-        }
+//        maven {
```

Comment:
Should we just delete this node? if needed in the future (to use snapshots) we can add it later (at the end there is work to do here regarding which build number you are checking, so there will be work to do).